action.yml

```diff
@@ -0,0 +1,38 @@
+name: 'Run quickstart'
+description: 'Runs quickstart docker image'
+inputs:
+  version:
+    description: "Version (docker tag) of quickstart image to use"
+    required: true
+    default: "latest"
+runs:
+  using: "composite"
+  steps:
+    - name: Setup docker (missing on MacOS)
+      shell: bash
+      if: runner.os == 'macos'
+      run: |
+        brew install docker
+        brew install colima
+        colima start
+    - run: |
+        docker run -d --name quickstart -p 8000:8000 -e ENABLE_LOGS=true -e NETWORK=local -e ENABLE_SOROBAN_RPC=true --health-cmd "curl --no-progress-meter --fail-with-body -X POST \"http://localhost:8000/soroban/rpc\" -H 'Content-Type: application/json' -d '{\"jsonrpc\":\"2.0\",\"id\":8675309,\"method\":\"getNetwork\"}' && curl --no-progress-meter \"http://localhost:8000/friendbot\" | grep '\"invalid_field\": \"addr\"'" --health-interval 10s --health-timeout 5s --health-retries 50 stellar/quickstart:${{ inputs.version }}
```

Comment:
ENABLE_SOROBAN was deprecaed at some point I thought, in favor of the ENABLE option where you list out the services. In local mode rpc always runs.